js/index.js

```diff
@@ -73,9 +96,16 @@ app.controller("IndexController", ["$scope", "$http", 'leafletData', function($s
       $scope.markers = response.data;
       var dateString;
       $($scope.markers).each(function(index, marker) {
-        dateString= new Date(marker.dueDate).toGMTString()
+        dateString = new Date(marker.dueDate).toGMTString()
         dateString = dateString.substring(0, dateString.length - 4);
         $scope.markers[index].message = "<b>Title: </b>" + marker.title + "<br/><b>Due Date: </b>" + dateString;
+        if(new Date(marker.dueDate) > new Date()){
```

Comment:
So if it's today, it's considered already in the past? Or are these really datetimes?